ci/platform-independence.yml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+steps:
+  - bash: ci/dev-env-install.sh
+    displayName: 'Build/Install the Developer Environment'
+  - bash: ci/configure-bazel.sh
+    displayName: 'Configure Bazel for root workspace'
+    env:
+      IS_FORK: $(System.PullRequest.IsFork)
+      # to upload to the bazel cache
+      GOOGLE_APPLICATION_CREDENTIALS_CONTENT: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
+  - task: DownloadPipelineArtifact@0
+    inputs:
+      artifactName: 'platform-independence-dalf'
+      targetPath: $(Build.StagingDirectory)/
+  - bash: |
+      set -euo pipefail
+      eval "$(dev-env/bin/dade-assist)"
+      bazel build //compiler/damlc/tests:platform-independence.dalf
```

Comment:
can we check the full DAR rather than just this DALF? If those are identical the dalfs are clearly identical as well and our DARs should be built deterministically (if not we should probably fix that)so I think there shouldn’t be extra-nondeterminism introduced here.

Reply:
The DARs are not identical: they include source code for a test file that differs on line endings, as well a .hie files that somehow differ too ("binary files differ", as `diff` puts it; I did not investigate further).